src/apps/persistence/jobs.clj

```diff
@@ -106,52 +109,39 @@
   [v]
   (if (zero? v) nil v))
 
-(defn- filter-value->where-value
-  "Returns a value for use in a job query where-clause map, based on the given filter field and
-   value pair."
-  [field value]
-  (case field
-    "app_name"  ['like (sqlfn :lower (str "%" value "%"))]
-    "name"      ['like (sqlfn :lower (str "%" value "%"))]
-    "id"        (when-not (string/blank? value) (uuidify value))
-    "parent_id" (when-not (string/blank? value) (uuidify value))
-    value))
-
-(defn- filter-field->where-field
-  "Returns a field key for use in a job query where-clause map, based on the given filter field."
-  [field]
-  (case field
-    "app_name" (sqlfn :lower :j.app_name)
-    "name"     (sqlfn :lower :j.job_name)
-    (keyword (str "j." field))))
-
-(defn- filter-map->where-clause
+(defn- hsql-filter-map->where-clause
```

Comment:
I was on the fence about removing the `hsql-` prefixes from these functions. I thought I might leave them in for now just to clearly indicate that these functions use HoneySQL rather than Korma.